client/docs/issues/settings-flat-style.md

```
@@ -0,0 +1,36 @@
+## 제목
+
+🎨 [기능개선][설정] Settings 페이지 flat 스타일 리디자인
```

markdownlint-cli2 (0.22.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "설정"

(MD052, reference-links-images)